packages/user/Sites/include/services/user/Sites.hpp

```diff
@@ -44,11 +44,11 @@ namespace SystemService
    struct SiteConfigRow
    {
       psibase::AccountNumber     account;
-      bool                       spa       = false;
```

Comment:
For types with a trivial default constructor, default initialization is not the same as zero initialization.

Reply:
Reverted